src/app/catalogue-search/profile-filter-dialog-component/profile-filter-dialog-component.component.html

```diff
@@ -0,0 +1,26 @@
+<!--
+Copyright 2020-2023 University of Oxford and NHS England
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<mat-dialog-content>
+  <mdm-catalogue-search-profile-filter-list
+    [prefilledFilters]="data.profileFilters"
+  ></mdm-catalogue-search-profile-filter-list>
+</mat-dialog-content>
+<mat-dialog-actions>
```

Comment:
Add `align="end"` to `<mat-dialog-actions>` to consistently align the buttons to the right.